fcgiwrap.c

```diff
@@ -578,6 +578,7 @@ static void handle_fcgi_request(void)
 			}
 
 			execl(filename, filename, (void *)NULL);
+			fprintf(stderr, "Error: %s\n", strerror(errno));
```

Comment:
I suggest to add a parameter to `cgi_error` that can be used to append additional information that should only be logged to the error log (instead of stdout). This can then be NULL for everything except the internal server error. What do you think of that?